src/main/webapp/i18n/en/iris.json

```
@@ -32,6 +32,19 @@
                     },
                     "hestiaSettings": "Hestia Settings",
                     "competencyGenerationSettings": "Competency Generation Settings",
+                    "proactivityBuildFailedEventEnabled": {
+                        "label": "Monitor submission build failures",
+                        "tooltip": "When enabled, Iris will proactively send help messages if a submission build fails."
+                    },
+                    "proactivityProgressStalledEventEnabled": {
+                        "label": "Monitor exercise performance progress",
+                        "tooltip": "When enabled, Iris will proactively send help messages if the student's exercise performance does not improve."
+                    },
+                    "proactivitySettings": {
+                        "title": "Proactivity Settings",
+                        "tooltip": "Enable options below to allow Iris to proactively reach out to students when specific conditions are met.",
+                        "parentDisabled": "Iris won't send proactive messages for this option because either option itself or Iris are disabled in the current or parent settings."
```

I think that's a lot more understandable now, thx :D only one small change request left:

```suggestion
                        "parentDisabled": "Iris won't send proactive messages for this option because either the option itself or Iris are disabled in the current or parent settings."
```

---

Addressed the change request